thirdeye/thirdeye-dashboard/src/main/java/org/apache/pinot/thirdeye/dashboard/resources/v2/RootCauseMetricResource.java

```diff
@@ -284,11 +288,54 @@ public double getAggregate(@ApiParam(value = "metric urn", required = true) @Que
     urns = ResourceUtils.parseListParams(urns);
     offsets = ResourceUtils.parseListParams(offsets);
     List<MetricSlice> slices = new ArrayList<>();
+    List<String> forecastOffsets = Arrays.asList(OFFSET_FORECAST, OFFSET_LOWER, OFFSET_UPPER);
+    Map<Pair<String, String>, Double> offsetToForecastAggregate = new HashMap<>();
+
+    if(offsets.contains(OFFSET_FORECAST)) {
```

Comment:
should this check for `OFFSET_LOWER` and `OFFSET_UPPER` as well?

Reply:
Good question.  At the moment, we are only serving upper and lower bounds in the case of forecast.  If someone requested the upper or lower bound and not the forecast, it would call the forecast endpoint (if I add upper and lower here).  Do we want that to happen?  As it's written now, it will populate upper and lower values with NaN if forecast isn't present.

Reply:
I see. It makes sense. We can keep it like this.